PICMI_Python/requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy~=1.15
 scipy~=1.5
+typeguard
```

Comment:
Checked: pure python (incl. dependencies) & easy to install it seems
- https://pypi.org/project/typeguard/#files
- https://github.com/agronholm/typeguard/blob/master/pyproject.toml
- https://github.com/agronholm/typeguard/blob/master/setup.cfg